Core/Inc/canardRxQueue.h

```diff
@@ -0,0 +1,18 @@
+#ifndef CANARD_RX_QUEUE_H
+#define CANARD_RX_QUEUE_H
+
+#include <stdlib.h>
+#include <canard.h>
+
+struct CanardRxQueueItem {
+    struct CanardRxQueueItem* next;
+    CanardCANFrame frame;
+};
+
```

Comment:
What is the size of the queue? If there is no boundary for the size of the queue, then what if the rx processing speed is slower than the rx incoming speed? I guess the system would run out of the memory and hard fault.

I think it will be better to keep track of the length of the queue somehow.